### PR TITLE
Dev ntu energy cluster agent

### PR DESCRIPTION
**Pull request by csl37**
Key changes:
-Updated energy cluster agent to use latest "Bubble Neural Network" from NTU and output voltage clusters for visualisation.
-Removed NN training. The trained model is now supplied during configuration.
-Added option to cluster OPF results.

### CODE REVIEW

File: Agents/NTUEnergyClusterAgent/README.md [hunk 1]
```diff
@@ -1,45 +1,35 @@
...
+- BNN with P2P: As above but this receives the power consumption values of peer-to-peer trading as input and uses a neural network to predict the voltage group of each bus in the power network. 
+- OPF: This method reads the voltage values for the network (calculated by the OPF Agent) from the KG and assigns voltage groups.
 
-For the agent to read data, three key components must be provided. The details of generating the key components are instructed in the following sections.
-- An NTU power system knowledge graph instantiated by the [HistoricalNTUEnergyAgent](https://github.com/cambridge-cares/TheWorldAvatar/tree/1496-dev-instantiate-historic-ntuenergyconsumptiondata-2/Agents/HistoricalNTUEnergyAgent).
-- A trained neural network model file in `.h5` format.
-- A k-means clustering model in `.pkl` format.
+The agent returns the voltage groups (0: under, 1: nominal, 2: over) intended for visualisation.
+
+For the agent to process opf results a power system must be instantiated by the [HistoricalNTUEnergyAgent](https://github.com/cambridge-cares/TheWorldAvatar/tree/1496-dev-instantiate-historic-ntuenergyconsumptiondata-2/Agents/HistoricalNTUEnergyAgent).
```
Comment (sganguly-cares) [inline Agents/NTUEnergyClusterAgent/README.md:10]: Update the URL for HistoricalNTUEnergyAgent

File: Agents/NTUEnergyClusterAgent/README.md [hunk 2]
```diff
@@ -1,45 +1,35 @@
...
 
 #### NTU Power System Knowledge Graph
-This knowledge graph is used to define the relationships between individual concepts in a power system and store both static and time-series readings to simulate a 13-bus power system at NTU. All values required to run this agent should be queried from the NTU Power System Knowledge Graph using the [time-series client](https://github.com/cambridge-cares/TheWorldAvatar/tree/develop/JPS_BASE_LIB/src/main/java/uk/ac/cam/cares/jps/base/timeseries) from the JPS_BASE_LIB to interact with both the knowledge graph and database.
 - For details to instantiate the NTU Power System Knowledge Graph, pleaes refer to the [HistoricalNTUEnergyAgent](https://github.com/cambridge-cares/TheWorldAvatar/tree/1496-dev-instantiate-historic-ntuenergyconsumptiondata-2/Agents/HistoricalNTUEnergyAgent).
```
Comment (sganguly-cares) [inline Agents/NTUEnergyClusterAgent/README.md:13]: Update the URL for HistoricalNTUEnergyAgent

**Review by sganguly-cares — COMMENTED** [review]
Some URLs in the readme need updating

**Comment by csl37** [comment]
URLs updated

**Review by sganguly-cares — APPROVED** [review]

**Pull request merged**